src/service/extension-location.js

```diff
@@ -57,12 +57,7 @@ export function calculateExtensionScriptUrl(location, extensionId,
   opt_extensionVersion, opt_isLocalDev) {
   const base = calculateScriptBaseUrl(location, opt_isLocalDev);
   const rtv = getMode().rtvVersion;
-  if (opt_extensionVersion == null) {
```

Comment:
Are you sure about this change?
I think we need to support the case where `opt_extensionVersion` equals the empty string and nothing is appended.

Is there no test for this?